value/src/test/java/com/google/auto/value/processor/AutoBuilderCompilationTest.java

```diff
@@ -647,7 +647,7 @@ public void alienOneArgMethod() {
     assertThat(compilation).failed();
     assertThat(compilation)
         .hadErrorContaining(
-            "[AutoBuilderBuilderWhatProp] Method does not correspond to a parameter of Baz(int one,"
+            "[AutoBuilderBuilderWhatProp] Method three does not correspond to a parameter of Baz(int one,"
```

Comment:
Can you please rewrap this so it doesn't go over 100 characters?

Reply:
Yep, updated